cg-spec/editors_draft.html

```diff
@@ -441,20 +441,25 @@ <h2>Structure of this document</h2>
     </section>
 
     <section id="conformance">
-      <p>A system supports RDF-star:</p>
-      <ul>
-        <li>if it supports one or more of the RDF 1.1 standard syntaxes for input, and 
-          one or more of its input syntaxes passes the tests described in the
-          <a href="https://w3c.github.io/rdf-star/tests/">RDF-star test suite</a>
-          (i.e., the <a href="https://w3c.github.io/rdf-star/tests/nt/syntax/">N-Triples Syntax tests</a>,
-          the <a href="https://w3c.github.io/rdf-star/tests/turtle/syntax/">Turtle Syntax tests</a>, or
-          the <a href="https://w3c.github.io/rdf-star/tests/trig/syntax/">TriG Syntax tests</a>).
-        </li>
-        <li>if it supports one or more of the RDF 1.1 standard syntaxes for output, and
-          one or more of its output syntaxes permits <a>quoted triples</a> as described in the
-          <a href="https://w3c.github.io/rdf-star/tests/">RDF-star test suite</a>.
-        </li>
-      </ul>
+      <p>A system supports RDF-star for input if it supports an input syntax for 
+        an RDF 1.1 standard syntax that has additional syntax for  
+        <a>quoted triples</a>.
+        Such input syntaxes include those in the 
+        <a href="https://w3c.github.io/rdf-star/tests/">RDF-star test suite</a>
+        (<a href="https://w3c.github.io/rdf-star/tests/nt/syntax/">N-Triples-star syntax tests</a>
+        , <a href="https://w3c.github.io/rdf-star/tests/turtle/syntax/">Turtle-star syntax tests</a>
+        , or <a href="https://w3c.github.io/rdf-star/tests/trig/syntax/">TriG-star syntax tests</a>).
```

Comment:
```suggestion
        (<a href="https://w3c.github.io/rdf-star/tests/nt/syntax/">N-Triples-star syntax tests</a>.
        <a href="https://w3c.github.io/rdf-star/tests/turtle/syntax/">Turtle-star syntax tests</a>,
        or <a href="https://w3c.github.io/rdf-star/tests/trig/syntax/">TriG-star syntax tests</a>).
```

Reply:
I find it easier managing the HTML text to have the comma at the start of line aligned to the "(". XML indenting works better (IMO).

Reply:
the problem is that this adds spurious whitespaces *before* the comma, which is not typographically kosher (at least I assume that's the reason for @TallTed's suggestion).
Another option is to push the closing `>` after the new line and indentation, just before the comma:
```html
    <a href="...">N-Triples-star syntax</a
    >,<a href="...">...
```
but that's quite ugly

Reply:
Oh - OK. Didn't understand that point.